### DISCUSSION TITLE
Revert context  menu

### DISCUSSION THREAD

**Discussion by StepanAsk** [Raphire/Win11Debloat#108] [Q&A]
How to revert win 11 context menu? or maybe there is a way to show it by user request?

**Comment by Raphire** [comment]
Heya,

The contextmenu can be reverted back to the W11 one by using this registry file that's provided with the script: https://github.com/Raphire/Win11Debloat/blob/master/Regfiles/Undo/Enable_Show_More_Options_Context_Menu.reg

I don't know of any way to temporarily show the W11 one, but you can immediately show the W10 one by shift-rightclicking.
